### DISCUSSION TITLE
Rename field in Prisma 2 without data loss

### DISCUSSION THREAD

**Discussion by OliverEvans96** [prisma/prisma#2323] [Q&A] [answered]
Hi all,

Just wondering if there's a prisma 2 equivalent for the temporary `@rename` directive for changing the name of a field without losing its data - as in https://github.com/prisma/prisma-docs-content-v2/blob/master/1.13/develop-prisma-service/workflows/migrations-asdf.mdx

Thanks,
Oliver

**Comment by ryands17** [comment, marked as answer]
Hey @OliverEvans96 :wave: 

Currently there's an open issue for this workflow to be implemented which you can track [here](https://github.com/prisma/prisma/issues/2033)

As a workaround for now, you would have to

1. Create a new field and migrate
2. Seed the data from the old to the new field
3. Delete the old field and migrate

Let me know if that helps!

**Reply by OliverEvans96** [reply]
Great, thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
